backend/wmg/data/cube_pipeline.py

```diff
@@ -28,7 +28,7 @@ def gen_pipeline_failure_message(failure_info: str) -> dict:
                 "type": "header",
                 "text": {
                     "type": "plain_text",
-                    "text": f"Corpus Asset Pipeline Failed:fire: \n{failure_info}",
+                    "text": f"WMG Corpus Asset Pipeline Failed:fire: \n{failure_info}",
```

Comment:
In the event of a failure, what should the on-call engineer check? Could we include a link to the logs here, for instance?